pre_commit/color.py

```diff
@@ -29,7 +30,7 @@ def format_color(text, color, use_color_setting):
         color - The color start string
         use_color_setting - Whether or not to color
     """
-    if not use_color_setting:
+    if not use_color_setting or not terminal_supports_colors:
```

Comment:
hmm, given the test failures, this should probably happen in the `use_color` function below and not in `format_color` (to keep this function pure and to keep `auto` / `always` functioning properly)

Probably change the the return to:

```python
return (
    setting == 'always' or
    (setting == 'auto' and sys.stdout.isatty() and terminal_supports_color)
)
```